scripts/storage-initializer/push_image.sh

```diff
@@ -17,4 +17,9 @@
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
 source build_image.sh
-docker push $IMAGE
+
+if [ ! -n "$1" ]; then
```

Comment:
Use environment variable instead of argument list to toggle cross build capability.

Reply:
Such as
```suggestion
if [ -z "${CROSS_PLATFORMS:-}" ]; then
```

Reply:
ok